include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
@evanlucas again, the introduction of new field could be easily avoided, if there is a way to merge two `if`s below. Are you sure this is not possible to do?